addons/pad/static/src/js/pad.js

```diff
@@ -12,7 +12,12 @@ var FieldPad = AbstractField.extend({
     content: "",
     events: {
         'click .oe_pad_switch': '_onToggleFullScreen',
+        'click': '_onClick',
```

Comment:
maybe extends `events` instead of re-defining the click (and override onKeydown to do nothing)